### PR TITLE
Update transcript save method

### PR DESCRIPTION
**Pull request by prakharrathi25**
# Description 

In the save endpoint, I have now added a check for a video, user, transcript_type and language. If a transcript is found then the payload is updated otherwise a new transcript is created. 

Another thing to check is if the language is being passed as `en` and not `English` because we have now set the format in Chitralekha. 

Response Request Format
image

### CODE REVIEW

File: backend/transcript/views.py [hunk 1]
```diff
@@ -383,12 +390,31 @@ def save_transcription(request):
...
+        
+        # If a transcript exists, update the payload
+        if transcript:
+            transcript.payload = transcribed_data
+            transcript.save()
+
+            return Response(
+                {"id": transcript.id, "data": transcript.payload},
+                status=status.HTTP_200_OK,
+            )
```
Comment (GokulNC) [inline backend/transcript/views.py:415]: Just a small refactoring suggestion, shall we make it like this?

```
if transcript:
    transcript.payload = transcribed_data
else:
    transcript = # new transcript
transcript.save()
return response
```

**Review by GokulNC — APPROVED** [review]
Thanks, looks good to me, we can merge!

File: backend/transcript/views.py (hunk 1 above)
Comment (prakharrathi25) [reply backend/transcript/views.py:415]: yeah sure

Comment (prakharrathi25) [reply backend/transcript/views.py]: Done

**Pull request merged**